subset/network/network_tests.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import time
+import sys
+
+name_of_tests = ['network.min_send', 'network.application.min_send', 'dhcp.long', 'ntp.update']
+
+arguments = sys.argv
+
+test_request = str(arguments[1])
+
+test_id = -1
+
+for x in range(0, len(name_of_tests) - 1):
+    if test_request == name_of_tests[x]:
+        test_id = x
+
+print('test_id=' + str(test_id))
+
+cap_pcap_file = str(arguments[2])
+#capture.pcap
+
+device_address = '127.0.0.1'
+
+if test_id == 0:
+    device_address = str(arguments[3])
+
+report_filename = 'report.txt'
+
+packets_in_report = 10
+
+tcpdump_display_all_packets = 'tcpdump -n src host ' + device_address + ' -r ' + cap_pcap_file
+tcpdump_display_udp_bacnet_packets = 'tcpdump -n udp dst portrange 47808-47809 ' + cap_pcap_file
+tcpdump_display_arp_packets = 'tcpdump -v arp -r ' + cap_pcap_file
+tcpdump_display_ntp_packets = 'tcpdump dst port 123 -r ' + cap_pcap_file
+
+tests = [tcpdump_display_all_packets, tcpdump_display_udp_bacnet_packets, tcpdump_display_arp_packets, tcpdump_display_ntp_packets]
+
+def shell_command_fb(command, wait_time, terminate_flag):
+    process = subprocess.Popen(command, universal_newlines=True, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    text = process.stdout.read()
+    retcode = process.wait()
+    time.sleep(wait_time)
+    if terminate_flag:
+        process.terminate()
+    if len(text) > 0:
+        return text
+
+pointer_list_line_end = []
+
+def find_char_pointer(value, find_text, pointer_list):
+    pointer = 0
+    while pointer != -1:
```

Comment:
why do you need this?  It never starts as -1, and if it returns -1 then you break? Isn't this just "while True:"?

Reply:
essentially while true now,I will update code.
pythons implementation of while loop doesnt work same way as java,

Reply:
How are they different -- I'm curious.  My understanding to date has been that they're pretty much the same (and the same as C) -- but maybe I missed something!

Reply:
I was not using global originally. case closed